build.sbt

```diff
@@ -91,4 +103,17 @@ lazy val cli = project.
     scalaVersion := scala212Version,
     // cli is not meant to be published
     publish / skip := true,
+    // We are creating a fatjar here for distribution so we can do global optimization
+    // within the scala.tools.jardiff package. If we implement a way to make the cli
+    // release in CI then we can also use insideCI
+    scalacOptions ++= Seq(
+      "-opt-inline-from:scala.tools.jardiff.**",
```

Comment:
You could even go for `-opt-inline-from:scala.**`

Reply:
Nice one, missed this! Doing it now

Reply:
Done and pushed.